Dockerfile

```
@@ -10,8 +10,7 @@ ARG TARGETARCH
 ENV ASAN_TAG=$ASAN_TAG
 ENV TARGETARCH=$TARGETARCH
 
-RUN	mkdir /deb
-COPY	--from=pkg /deb/* /deb/
+COPY	--from=pkg /deb /deb
```

> alternative approach may be something like `RUN --mount` but I haven't tried it

Should look like this:

```suggestion
RUN --mount=type=bind,from=pkg,source=/deb,target=/deb <<EOF
  apt-get update
  dpkg -i /deb/rspamd${ASAN_TAG}_*_${TARGETARCH}.deb /deb/rspamd${ASAN_TAG}-dbg_*_${TARGETARCH}.deb || true
  apt-get install -f -y
  apt-get -q clean
  dpkg-query --no-pager -l rspamd${ASAN_TAG}
  rm -rf /var/cache/debconf /var/lib/apt/lists
  rm -rf /deb
EOF
```

---

Note these features [requires BuildKit with Docker release since 2022H2](https://github.com/oven-sh/bun/pull/6100#issue-1915168318) (_preferably 2023 release_). You'll need to [add this syntax hint](https://github.com/docker-mailserver/docker-mailserver/blob/bd96c1161e9f9b32bb0b3aaec4b930980a56945f/Dockerfile#L1) for older Docker builders IIRC.